README.md

```diff
@@ -1,2 +1,175 @@
-# Hello-Microverse
-Set up a "Hello Microverse" project Exersise
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+</div>
```

Comment:
- [ ] Kindly remove the microverse logo. This is a placeholder. You can add your own logo or leave it blank if you don't have a logo.